src/ContinuousEnergyNeutron.tex

```diff
@@ -222,37 +242,74 @@ \subsubsection{\textsf{LQR} Block}\label{sec:LQRBlock}
   \label{tab:LQRBlock}
 \end{BlockTable}
 
-\subsubsection{\textsf{TYR} Block}\label{sec:TYRBlock}
-The format of the \Block{TYR} is given in \Tableref{tab:TYRBlock}. The index at the start of the \Block{TYR}, \startblock{TYR}=\jxs{5}. The number of reactions, \var{NMT}, is the same through the \ACE\ Table, \var{NMT}=\nxs{4}.
+\subsubsection{\textsf{TYR} and \textsf{TYRH} Block}\label{sec:TYRBlock}\label{sec:TYRHBlock}
+
+The format of the \Block{TYR} (for incident neutron reactions) and \Block{TYRH} (for particle production reactions) is given in \Tableref{tab:TYRBlock}. The starting index \var{LTYR} depends on whether it is the \Block{TYR} or \Block{TYRH} and is given in \Tableref{tab:TYR_NMT}. For the particle production \Block{TYRH}, \var{i} refers to the index of the corresponding particle type defined on the \Block{PTYPE} and is between 1 and \var{NTYPE}.
+
+\begin{table}[h!] \centering
+  \begin{tabular}[h]{lll}
+    \toprule
+    Block      & \var{LTYR}                 & \var{NMT} \\
+    \midrule
+    \var{TYR}  & \jxs{3}                   & \nxs{4} \\
```

Comment:
The original MCNP5 manual version says the LTYR=JXS(3) for the TYR Block. Was this intentionally changed or is this a mistake?